Purchases/Public/RCPurchases.h

```diff
@@ -130,6 +130,26 @@ NS_SWIFT_NAME(Purchases)
 + (instancetype)configureWithAPIKey:(NSString *)APIKey
                           appUserID:(NSString * _Nullable)appUserID
                        userDefaults:(NSUserDefaults * _Nullable)userDefaults;
+
+
+/**
+ Configures an instance of the Purchases SDK with object with a custom userDefaults. Use this constructor if you want to sync status across a shared container, such as between a host app and an extension. The instance of the Purchases SDK will be set as a singleton. You should access the singleton instance using [RCPurchases sharedPurchases]
+ 
+ @param APIKey The API Key generated for your app from https://app.revenuecat.com/
+ 
+ @param appUserID The unique app user id for this user. This user id will allow users to share their purchases and subscriptions across devices. Pass nil if you want `RCPurchases` to generate this for you.
+ 
+ @param userDefaults Custom userDefaults to use
+ 
+ @param observerMode Set this to TRUE if you have your own subscription system and want to use only RevenueCat's backend. Default is FALSE.
```

Comment:
"subscription system" might not be the right name. 
If you have your own "IAP implementation" might be better.

@rkotzy thoughts?

Reply:
can we change this to "IAP implementation"?